src/PrestissimoFileFetcher.php

```diff
@@ -43,7 +37,16 @@ protected function fetchWithPrestissimo($version, $destination) {
     array_walk($this->filenames, function ($filename) use ($version, $destination, &$requests) {
       $url = $this->getUri($filename, $version);
       $this->fs->ensureDirectoryExists($destination . '/' . dirname($filename));
-      $requests[] = new CopyRequest($url, $destination . '/' . $filename, false, $this->io, $this->config);
+      if ($this->progress) {
```

Comment:
I think, we can skip this hunk and add the information in line 65.

Reply:
Seen with @greg-1-anderson. Will update the pull request.